.github/workflows/ci-update_trading.yml

```diff
@@ -12,6 +12,8 @@ jobs:
         run: |
           cd trading_report
           yarn install
+          SYMBOL=DUAL RPC_URL=$RPC_URL yarn run main
+          mv transactions.csv ../app/public/sol_transactions.csv;  mv offers.csv ../app/public/sol_offers.csv
```

Comment:
Do you want dual_transacations and dual_offers?

Reply:
also, check that the constant in line 24 doesnt need to change

Reply:
Leaving offers. Added file length

Reply:
Is line 22 going to overwrite this result?

Reply:
Running trading report here

Reply:
Why? It doesnt change any permanent state and you are overwriting the saved results in line 22

Reply:
why not dual_transactions and dual offers?